src/site/stages/build/process-cms-exports/schemas/raw/media-document.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable camelcase */
+
+module.exports = {
+  type: 'object',
+  properties: {
+    name: { $ref: 'GenericNestedString' },
+    path: {
+      type: 'array',
+      maxItems: 1,
+      items: {
+        type: 'object',
+        properties: {
+          alias: { type: 'string' },
```

Comment:
:thinking: So right now, we require this to be a string because it's how we find the file (document) this entity is for and without it, we have no other way. Right?

If we merge this in, will `npm run build:content:test` finish successfully? If not, we may need to make it nullable for now, and write a ticket to address it when we have the ability to get the URL of the file instead of relying on this alias.

Reply:
Yes to your first question.

No to your second - `build:content:test:` will fail.

So, with this change the tests & content build will pass:

![image](https://user-images.githubusercontent.com/2008881/71853264-aa123d00-308f-11ea-8ddc-6723bad306eb.png)


I suppose I'll commit them and then write up an issue for the url, and maybe open a PR for `nullableString` as well :slightly_smiling_face: